### DISCUSSION TITLE
How to remove Ctrl + Shift + - shortcut?

### DISCUSSION THREAD

**Discussion by eliasallopes** [dbeaver/dbeaver#35315] [Q&A] [answered]
Hello, 

While usind DBeaver I have come across the shortcut  Ctrl + Shift + - that seems to open the same code on a different window. 
I want to disable this shortcut, because is something I never use on purpose, but sometimes accidentally hit and, I don't how to undo what it does (If you know how, I would love to know that also).
But when I go to shortcut editor I can't find this shortcut.
Is there are a way to disable it?

Thank you in advance

**Comment by TB-HunBoka** [comment, marked as answer]
Hello @eliasallopes 
You can't find it because only part of the shortcut is what you are looking for. The shift key is not part of the key however that's changes your **-** to **_** .
The key you are looking for is ctrl+_ Toggle Split Editor (horizontal)
You can undo this by hitting the same ctrl+shift+- again or by clicking the window->editor->Toggle Split Editor
